lib/util/customConfig.js

```diff
@@ -14,8 +14,8 @@ try {
 const CHECK_REFRESH_RATE = 1_000;
 let previousConfig = null;
```

Comment:
@hyoban I guess `previousConfig` should not be needed anymore if we keep a cache of all the configs we find?

also this part

```
const newConfig = convertConfigToString(twConfig) !== convertConfigToString(previousConfig);
```

it's probably also not necessary and we could check if the config is in the cache?

Reply:
Yes, this is better, thanks for your review!

Reply:
The code has been updated to apply your suggestion